sirepo/job_supervisor.py

```diff
@@ -414,10 +478,15 @@ def _ops_to_cancel():
                     await o.prepare_send()
                     self.run_op = o
                     self.__db_init(req, prev_db=self.db)
-                    # run mode can change between runs so we must update the db
-                    self.db.jobRunMode = req.content.jobRunMode
-                    self.db.computeJobSerial = int(time.time())
-                    self.db.pkupdate(status=job.PENDING)
+                    self.db.pkupdate(
+                        computeJobSerial=int(time.time()),
+                        computeModel=req.content.computeModel,
+                        driverDetails=o.driver.driver_details,
+                        # run mode can change between runs so we must update the db
+                        jobRunMode=req.content.jobRunMode,
+                        name=req.content.data.models.simulation.name,
```

Comment:
name => simName. It's easier to see what it is when it is in isolation. We probably want the simName in the history as it may change.